openc3/lib/openc3/microservices/trigger_group_microservice.rb

```diff
@@ -266,38 +271,48 @@ def initialize(name:, logger:, scope:, group:, queue:, share:, ident:)
       @ident = ident
     end
 
+    def notify(name:, severity:, message:)
+      data = {}
+      # All AutonomicTopic notifications must have 'name' and 'updated_at' in the data
+      data['name'] = name
+      data['updated_at'] = Time.now.to_nsec_from_epoch
+      data['severity'] = severity
+      data['message'] = message
+      notification = {
+        'kind' => 'error',
+        'type' => 'trigger',
+        'data' => JSON.generate(data),
+      }
+      AutonomicTopic.write_notification(notification, scope: @scope)
+      @logger.public_send(severity.intern, message)
+    end
```

Comment:
I created this to notify both the AutonomicTopic (so the Autonomic frontend can show errors) as well as the logger which updates the LogMessages. It's sort of double booking it so perhaps it should only notify the AutonomicTopic? I only do this if there is an error.